job-dsls/jobs/kie/main/srcclr_scan_pipeline.groovy

```diff
@@ -84,4 +84,14 @@ pipelineJob("${folderPath}/srcclrpipeline") {
         }
     }
 
+    properties {
+        pipelineTriggers {
+            triggers {
+                cron {
+                    spec('0 23 14,28 * *')
```

Comment:
1 and 15 are also valid days (probably more user friendly :thinking: ) I'm ok with both 
```suggestion
                    spec('0 23 1,15 * *')
```